smart_pet_buddy/lib/constants.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+//for animated text
+const colorizeColors = [
+  Colors.green,
+  Colors.pink,
+  Colors.yellow,
+  Colors.white,
+];
+const colorizeTextStyle = TextStyle(
+    fontSize: 30.0,
+    fontFamily: 'GoodDog New',
+    fontWeight: FontWeight.bold
+);
+
+
+
+//color scheme
+const lightPrimary = Color(0xFF9FDBE7);
+const strongPrimary = Color(0xFF005263);
+const textColor = Color(0xFF020122);
+const midPrimary = Color(0xFF62A8AC);
+const midShade = Color(0xFFFFC65C);
+const strongShade = Color(0xFFE57F17);
+const lightShade = Color(0xFFF5DFA0);
+
+//text
+const primaryTextFont = TextStyle(fontFamily: 'Nexa Rust');
```

Comment:
I think it is a good idea to have these 👍

Reply:
> I think it is a good idea to have these 👍

Yes! We can quickly adjust colours, just changing it here